server/controllers/admin.controller.js

```diff
@@ -266,3 +266,101 @@ function getNewConfig(datas, settableConfig) {
   }
   return newConfig;
 }
+
+export function listServers(req, res) {
+  return Server.find()
+    .then((servers) => {
+      let authServer;
+      const otherServers = [];
+
+      for (let i = 0; i < servers.length; i += 1) {
```

Comment:
Au lieu de devoir faire une itération avec tous les serveurs, tu peux créer un `Promise.all()` avec deux requêtes de BDD - une pour chercher le serveur authoritative, et une autre pour tous les autres serveurs, puis de les donner à `res.render()` comme `authoritativeServer` et `otherServers`, respectivement. Puis dans le gabarit tu auras aussi un meilleur contrôle de comment tu les affiches.